planning/autoware_costmap_generator/nodes/autoware_costmap_generator/costmap_generator_node.cpp

```diff
@@ -277,7 +279,13 @@ void CostmapGenerator::onScenario(const tier4_planning_msgs::msg::Scenario::Cons
 void CostmapGenerator::onTimer()
 {
   autoware::universe_utils::ScopedTimeTrack scoped_time_track(__func__, *time_keeper_);
+  stop_watch.tic();
   if (!isActive()) {
+    // Publish ProcessingTime
+    tier4_debug_msgs::msg::Float64Stamped processing_time_msg;
+    processing_time_msg.stamp = get_clock()->now();
+    processing_time_msg.data = stop_watch.toc();
+    pub_processing_time_ms_->publish(processing_time_msg);
```

Comment:
memo:
I understood that processing time must always be publised, even if the process does not run.
This is for scenario testing and graph creation.